plugin.py

```diff
@@ -13,5 +13,4 @@ def plugin_unloaded():
 class LspTailwindcssPlugin(NpmClientHandler):
     package_name = __package__
     server_directory = 'language-server'
-    server_binary_path = os.path.join(server_directory, 'server', 'tailwindServer.js')
-    skip_npm_install = True
+    server_binary_path = os.path.join(server_directory, 'node_modules', '.bin', 'tailwindcss-language-server')
```

Comment:
Is this what you meant https://github.com/sublimelsp/LSP-tailwindcss/pull/38/commits/a6efc3f99e979092c6637826e20a0a903551c074?

Reply:
Seems like this server does things differently from what I've anticipated so it's still a shell script but it might still be better for Windows as the one in `.bin` will have a .cmd extension there which I believe was causing issues.